main.nf

```diff
@@ -1593,26 +1596,26 @@ process samtools_filter {
         '''
     } else if ( "${params.bam_unmapped_type}" == "bam" && params.bam_filter_minreadlength != 0 ){
         '''
-        samtools view -h !{bam} | samtools view - -@ !{task.cpus} -f4 -o !{libraryid}.unmapped.bam
-        samtools view -h !{bam} | samtools view - -@ !{task.cpus} -F4 -q !{params.bam_mapping_quality_threshold} -o tmp_mapped.bam
+        samtools view -h !{bam} | samtools view - -@ !{task.cpus - 1} -f4 -o !{libraryid}.unmapped.bam
```

Comment:
Why is `samtools view` here piped to `samtools view` again?

Reply:
Agreed, one could just run samtools view once with the `-h`flag and would get the same simplified 👍🏻

Reply:
Huh... no clue where that is coming from. will fix before merging!